priv/maker-ui/en.json

```diff
@@ -155,11 +155,9 @@
     "hub": "Hub"
   },
   "hubs_people": {
-    "change_role": "Change Role",
     "changing_role_admin_to_contributor": "Changing your role to a Collaborator will make it so you can't change others' roles.",
     "changing_role_owner_to_admin": "Changing your role to an Admin will make it so you can’t see billing info.",
     "changing_role_owner_to_contributor": "Changing your role to a Collaborator will make it so you can’t see billing info, or change others' roles.",
```

Comment:
Since they are part of the same component, can we also put that prefix on these too?